doc/Developing.md

```diff
@@ -73,6 +73,16 @@ variable in your shell:
 
     THEIA=$PWD/theia
 
+## Repository structure
+
+Theia repository has multiple folders:
```

Comment:
I chose the word "folder" to avoid confusion with the folder named `packages`

Reply:
the section looks good to me, I've added it to the index